release-notes/cabal-install-3.8.0.20220526.md

```diff
@@ -164,40 +163,68 @@ cabal-install 3.8.0.20220526 Changelog
   - Command-line `ghc-options` only applies to local packages
   - `program-options` stanza only applies to local packages
 
-- Improve error message for empty --allow-newer=  [#7740](https://github.com/haskell/cabal/issues/7740) [#8140](https://github.com/haskell/cabal/pull/8140)
+- Improve error message for empty `--allow-newer=`  [#7740](https://github.com/haskell/cabal/issues/7740) [#8140](https://github.com/haskell/cabal/pull/8140)
 
   Instead of internal error, the message now explains that empty argument for
-
+  `--allow-newer=` is not allowed and reminds what `--allow-newer` (with the empty
```

Comment:
This line was somehow lost when the changelog was assembled, I restored it from the PR.